spec/models/settings/smtp_spec.rb

```diff
@@ -1,6 +1,10 @@
 require "rails_helper"
 
 RSpec.describe Settings::SMTP do
+  after do
+    described_class.clear_cache
```

Comment:
this is new and solves another related problem?

Reply:
@djuber yep the settings here were bleeding into another spec.